<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1011</width>
      <height>821</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 53, 65);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>661</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;
font: 87 12pt &quot;Arial Black&quot;;
border-style: outset;
color: rgb(209, 209, 209);
</string>
      </property>
      <property name="text">
       <string>PROGRAMA DE AUTOMACAO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>170</y>
        <width>361</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;
font: 87 12pt &quot;Arial Black&quot;;
border-style: outset;
color: rgb(209, 209, 209);
</string>
      </property>
      <property name="text">
       <string>AINDA EM TESTE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pagNotas">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>991</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>18</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;
font: 87 18pt &quot;Arial Black&quot;;
border-style: outset;
color: rgb(209, 209, 209);
</string>
      </property>
      <property name="text">
       <string>LER NOTAS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNotatransformar">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>760</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
	background-color: rgb(7, 0, 235);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
	background-color: rgb(87, 255, 92);
font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>TRANSFORMAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNotaarq">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>110</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SELECIONAR PASTA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>851</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;
font: 87 12pt &quot;Arial Black&quot;;
border-style: outset;
color: rgb(209, 209, 209);
</string>
      </property>
      <property name="text">
       <string>SELECIONAR DOS ARQUIVOS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>841</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;
font: 87 12pt &quot;Arial Black&quot;;
border-style: outset;
color: rgb(209, 209, 209);
</string>
      </property>
      <property name="text">
       <string>LOCAL PARA SALVAR</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNotalocal">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>320</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SELECIONAR LOCAL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="localNotas">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>841</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtinfoNota">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>781</width>
        <height>441</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAbrPastaNota">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>710</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
	background-color: rgb(7, 0, 235);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
	background-color: rgb(87, 255, 92);
font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>ABRIR PASTA</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="localNotasSalvar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>841</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;
font: 87 12pt &quot;Arial Black&quot;;
border-style: outset;
color: rgb(209, 209, 209);
</string>
      </property>
      <property name="text">
       <string>TXT TOMADOS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>170</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;
font: 87 12pt &quot;Arial Black&quot;;
border-style: outset;
color: rgb(209, 209, 209);
</string>
      </property>
      <property name="text">
       <string>TXT ENTRADA</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtTomados">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtEntrada">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>210</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCNPJ">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>380</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
	background-color: rgb(7, 0, 235);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
	background-color: rgb(87, 255, 92);
font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>PESQUISAR CNPJS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_exportar">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>430</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
	background-color: rgb(7, 0, 235);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
	background-color: rgb(87, 255, 92);
font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>EXPORTAR BANCO</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pagPRN">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>991</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>18</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;
font: 87 18pt &quot;Arial Black&quot;;
border-style: outset;
color: rgb(209, 209, 209);
</string>
      </property>
      <property name="text">
       <string>Transformar PRN</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPRNarq">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>130</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SELECIONAR ARQ</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textoPRNarq">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>841</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="textoPRNlocal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>831</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPRNlocal">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>230</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SELECIONAR LOCAL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>981</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;
font: 87 12pt &quot;Arial Black&quot;;
border-style: outset;
color: rgb(209, 209, 209);
</string>
      </property>
      <property name="text">
       <string>LOCAL PARA SALVAR</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>991</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;
font: 87 12pt &quot;Arial Black&quot;;
border-style: outset;
color: rgb(209, 209, 209);
</string>
      </property>
      <property name="text">
       <string>SELECIONAR ARQUIVO/PASTA</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPRNtransformar">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>760</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
	background-color: rgb(7, 0, 235);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
	background-color: rgb(87, 255, 92);
font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>TRANSFORMAR</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txtinfoPRN">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>771</width>
        <height>511</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAbrPasta">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>710</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
	background-color: rgb(7, 0, 235);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
	background-color: rgb(87, 255, 92);
font-size: 25px;
}</string>
      </property>
      <property name="text">
       <string>ABRIR PASTA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPRNpasta">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>90</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SELECIONAR PASTA</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pagRazao">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>481</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;
font: 87 12pt &quot;Arial Black&quot;;
border-style: outset;
color: rgb(209, 209, 209);
</string>
      </property>
      <property name="text">
       <string>AINDA NAO FEITO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pagRateio">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;
font: 87 12pt &quot;Arial Black&quot;;
border-style: outset;
color: rgb(209, 209, 209);
</string>
      </property>
      <property name="text">
       <string>NAO FEITO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnNotas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color: rgb(48, 141, 255);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color:rgb(112, 169, 255);
font-size: 25px;
}</string>
    </property>
    <property name="text">
     <string>NOTAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPRN">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color: rgb(48, 141, 255);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color:rgb(112, 169, 255);
font-size: 25px;
}</string>
    </property>
    <property name="text">
     <string>PRN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRazao">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color: rgb(48, 141, 255);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color:rgb(112, 169, 255);
font-size: 25px;
}</string>
    </property>
    <property name="text">
     <string>RAZAO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRateio">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color: rgb(48, 141, 255);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color:rgb(112, 169, 255);
font-size: 25px;
}</string>
    </property>
    <property name="text">
     <string>RATEIO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVoltar">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>0</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
background-color: rgb(0, 255, 0);
font-size: 20px;
}
QPushButton:hover{
border-radius: 15px;
border-width:2px;
border-style:outset;
border-color:black;
	background-color: rgb(87, 255, 92);
font-size: 25px;
}</string>
    </property>
    <property name="text">
     <string>VOLTAR</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
